src/components/dashboard/menu-selector/menu-selector.js

```diff
@@ -2,30 +2,35 @@ import React, { useState, useEffect, useLayoutEffect } from 'react';
 
 import styled from "styled-components"
 
+import { MenuCreator } from "../menu-creator/menu-creator"
+
 import Client from "../../../util/client"
 
-let StyledMenuSelector = styled.h1`
+let StyledMenuSelector = styled.div`
     text-transform: uppercase;
     font-size: 24px;
     line-height: 12px;
-    font-family: HK Grotesk;
+    font-family: HK Grotesk regular;
     font-style: normal;
     font-weight: bold;
     font-feature-settings: 'cpsp' on;
     letter-spacing: 0.1em;
+    width: 100%;
+    padding-bottom: 20px;
 `
 
 const Menus = styled.div`
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
+    .menu-creator {
+        display: inline-block;
+        width: auto;
+    }
 `
 
 const MenuTab = styled.div`
+    display: inline-block;
```

Comment:
Could we increase the spacing between the lines?
![Screen Shot 2020-08-29 at 4 12 11 PM](https://user-images.githubusercontent.com/24697473/91647706-a6fb0100-ea12-11ea-857f-a1d2260a95c2.png)

Reply:
![image](https://user-images.githubusercontent.com/31301908/91651089-f22a0980-ea3c-11ea-95d4-1598f05531b7.png)

Is this better? If so, resolved!